e2e/cypress/plugins/index.js

```diff
@@ -1,17 +1,9 @@
-// ***********************************************************
-// This example plugins/index.js can be used to load plugins
-//
-// You can change the location of this file or turn off loading
-// the plugins file with the 'pluginsFile' configuration option.
-//
-// You can read more here:
-// https://on.cypress.io/plugins-guide
-// ***********************************************************
+const cypressTypeScriptPreprocessor = require('./cy-ts-preprocessor')
```

Comment:
Would like to see a comment at the top of the file:
```
// This file was generated when running the js-to-ts conversion.
// See https://github.com/bahmutov/add-typescript-to-cypress
```

Did that conversion also create the above file cy-ts-preprocessor.js ?
If so please add the same comment there.
If not, would be good to know where those settings came from.